src/app/example_configs/config.yaml

```diff
@@ -1,19 +1,19 @@
 auth:
-  enabled: false
```

Comment:
A bunch of stuff moved around here, I'm guessing because you are using the example config directly in your environment and it got rewritten when the application started.  You should clean this up so that the only diff is the thing that actually changed.